templates/definition/tariff/elering.yaml

```
@@ -12,7 +12,8 @@ group: price
 params:
   - name: region
     example: ee
-    validvalues: ["ee", "lt", "lv", "fi"]
+    type: choice
+    choice: ["ee", "lt", "lv", "fi"]
```

Ich würd hier auf die Mehrzahl gehen. Singular Var-Name mit Listeninhalt ist unerwartet.

```suggestion
    choices: ["ee", "lt", "lv", "fi"]
```

---

@naltatis ich würde es jetzt tatsächlich so lassen da das in fast jedem Tempalte auftaucht und schon immer so war...